coldfront/core/project/views.py

```diff
@@ -1075,53 +1075,64 @@ def post(self, request, *args, **kwargs):
         project_user_pk = self.kwargs.get('project_user_pk')
 
         if project_obj.status.name not in ['Active', 'New', ]:
-            messages.error(
-                request, 'You cannot update a user in an archived project.')
-            return HttpResponseRedirect(reverse('project-user-detail', kwargs={'pk': project_user_pk}))
+            messages.error(request, 'You cannot update a user in an archived project.')
+            return HttpResponseRedirect(reverse('project-user-detail', kwargs={'pk': project_obj.pk, 'project_user_pk': project_user_obj.pk}))
```

Comment:
`project_user_obj` isn't defined at this point--the old redirection seems appropriate to me.